server/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -92,6 +94,7 @@ public Content render(Params params) {
     bundle.addFooterScripts(layout.viewUtils.makeLocalJsTag("validation"));
 
     return layout.renderWithNav(
+        // TODO(clouser): Make sure applicant name is resolved.
```

Comment:
issue (blocking): please expand on this or remove

Reply:
oops, that's fixed. Removing.